docs/source/flow_setup.rst

```diff
@@ -58,10 +58,21 @@ We begin by downloading SUMO's github directory:
     git checkout 1d4338ab80
     make -f Makefile.cvs
 
-If you have OSX, run the following commands
+If you have OSX, run the following commands. If you don't have brew
+you can find installation instructions at 
+<https://docs.brew.sh/Installation>
 
 ::
-
```

Comment:
can u add this line again? (click view to understand why, it doesn't represent it as commands)